code/game/objects/items/devices/remotecyborg.dm

```diff
@@ -0,0 +1,296 @@
+/*
+* MODULE: Syndicate R.C. (Syndicate Remote Cyborg)
+* Description: This .dm adds the following robotics traitor items: cyborg camera bugs, cyborg control boards and the syndicate remote controller
+* Cyborg Camera Bugs: You can attach them to any silicon lifeform (AI, Cyborgs, pAI). It will allow you to see the user's screen almost one to one.
+* If the user has some kind of special vision (security hud for exemple), you will not have access to that.
+* Cyborg Control Boards: To use, you install this instead of a MMI when making a robot. It will create a cyborg as normal, but without a player to control it.
+* With the syndicate controller, you can control the cyborg like any other player would, selecting a module, able to change your name, communicate with the AI.
+* At the same time, you are linked to the cyborg consoles. So they can lock you down, and self-destruct the robot
+* Syndicate Controller: Imagine a controller with a screen and a keyboard, you can see the borgs you put the cameras on, or control your own borg.
+*
+* Other things:
+* Syndieborg: Just a borg that has the Cyborg Control Board. I made this so I could overwrite the login message, it doesn't make sense to have the player recieve the
+* law messages any time he connects to the borg. So it now displays a message about how to log off from the borg.
+* Frequency: Every object in this list has a frequency. You can make your own frequency by editing the variables. They all default to 'syndicate'.
+* What it means is, only boards on the syndicate frequency can communicate with syndicate controllers. You can have your own 'network', if you could say so.
+* Active: Just a 0 or 1 flag to state if the object (Camera or Control board) is online. Setting it to 0 will not instantly disconnect the player from the camera or borg though.
+* It is just a flag for when making the list. It updates when the mob with the camera, or board, dies.
+* Logoff Procedure: A small procedure made for the syndieborgs. It disconnects the player from the borg, allowing them to simply come back to their original body. Nice and easy.
+* 'Fuck Up Procs': It's the verifiers when the user, who is controlling or watching the camera, gets fucked up
+* (controller removed from his hands, downed, pushed, stunned, monkified, or anything that can disable him). It stops the user from viewing the camera,
```

Comment:
Proc should also be triggered if hiding in a locker. The user must be outside and visible to use it, or he can get away with anything the borg does without being implicated.